public/catalog/v0.1.0/openapi.json

```diff
@@ -207,9 +207,9 @@
           "users",
           "admins"
         ],
-        "summary": "Fetch all portfolio items from a specific portfolio",
+        "summary": "Fetch all items from a specific portfolio",
```

Comment:
@cbudz I'm wondering if removing the word 'portfolio` here makes the term 'items' too ambiguous?  Your change below leaves the term 'portfolio' in the description, so it seems apropos here as well.

Reply:
@syncrou thanks for catching that Drew. I'm in process of making a second pass for some style issues, but I'll note those updates in my next commit.